File: backend-php/api/model/Pessoa.php
<?php

/**
 * @author Matheus
 */
class Pessoa {
   
    private $nome;
    private $cpf;
    private $dataNascimento;
    private $endereco;
    private $email;
    private $telefone;
   
    public function get_cpf() {
        return $this->cpf;
    }

    public function get_nome() {
        return $this->nome;
    }

    public function get_dataNascimento() {
        return $this->dataNascimento;
    }
    public function get_endereco() {
        return $this->endereco;
    }

    public function get_email() {
        return $this->email;
    }

    public function get_telefone() {
        return $this->telefone;
    }
    
    public function set_nome($nome) {
        $this->nome = $nome;
        return $this;
    }

    public function set_cpf($cpf) {
        $this->nome = $cpf;
        return $this;
    }

    public function set_dataNascimento($dataNascimento) {
        $this->dataNascimento = $dataNascimento;
        return $this;
    }

    public function set_endereco($endereco) {
        $this->endereco = $endereco;
        return $this;
    }

    public function set_email($email) {
        $this->email=$email;
        return $this;
    }

    public function set_telefone($telefone) {
        $this->telefone=$telefone;
        return $this;
    }
}

File: AdotePetJavaProjetct/src/fabrica/Pessoa.java
package fabrica;

import java.util.Date;

public class Pessoa {
    private String nome;
    private String cpf;
    private Date dataNascimento;
    private String endereco;
    private String email;
    private String telefone;


    public Pessoa(String nome, String cpf, String endereco, String email, String telefone){
        setNome(nome);
        setCpf(cpf);
        setEndereco(endereco);
        setEmail(email);
        setTelefone(telefone);
    }


    public String getNome() {
        return nome;
    }

    public void setNome(String nome) {
        this.nome = nome;
    }

    public String getCpf() {
        return cpf;
    }

    public void setCpf(String cpf) {
        this.cpf = cpf;
    }

    public String getEndereco() {
        return endereco;
    }

    public void setEndereco(String endereco) {
        this.endereco = endereco;
    }

    public String getEmail() {
        return email;
    }

    public void setEmail(String email) {
        this.email = email;
    }

    public String getTelefone() {
        return telefone;
    }

    public void setTelefone(String telefone) {
        this.telefone = telefone;
    }

    public Date getDataNascimento() {
        return dataNascimento;
    }

    public void setDataNascimento(Date dataNascimento) {
        this.dataNascimento = dataNascimento;
    }
}
